add: useCache to destinationOptions of SAP Cloud SDK (#566)
cap-js/docs: guides/using-services.md

The `selectionStrategy` property controls how a [destination is resolved](#destination-resolution).

> The `useCache` option controls whether the SAP Cloud SDK caches the destination. Please see the [SAP Cloud SDK documenation](https://sap.github.io/cloud-sdk/docs/js/features/connectivity/destination-cache#destination-cache) for more information how the cache works.

Reviewer: ```suggestion
The `useCache` option controls whether the SAP Cloud SDK caches the destination. Read [Destination Cache](https://sap.github.io/cloud-sdk/docs/js/features/connectivity/destination-cache#destination-cache) to learn more about how the cache works.
```
